gitops/eshop/tsb-config.yaml

```diff
@@ -116,24 +116,25 @@ items:
           - route:
               host: gitops-eshop/products.gitops-eshop.svc.cluster.local
               port: 80
-        # rateLimiting:
-        #   settings:
-        #     rules:
-        #       - dimensions:
-        #           - remoteAddress:
-        #               value: '*'
-        #         limit:
-        #           requestsPerUnit: 3
-        #           unit: SECOND
-      # waf:
-      #   rules:
-      #     - Include @recommended-conf         # Recommended basic setup
-      #     - SecRuleEngine On                  # Override DetectionOnly mode in @recommended-conf
-      #     - SecResponseBodyAccess Off         # Override to not process response bodies
-      #     - Include @crs-setup-conf           # Initialize the Core Rule Set
-      #     - Include @owasp_crs/*.conf         # Load the Core Rule Set
-      #     # Custom rule to deny requests with query parameter 'id=0'
-      #     - SecRule ARGS:id "@eq 0" "id:1, phase:1,deny, status:403,msg:'Invalid id',log,auditlog"
+        rateLimiting:
+          settings:
+            rules:
+              - dimensions:
+                  - remoteAddress:
+                      value: '*'
+                limit:
+                  requestsPerUnit: 3
+                  unit: SECOND
+      waf:
+        rules:
+          - Include @recommended-conf         # Recommended basic setup
+          - SecRuleEngine On                  # Override DetectionOnly mode in @recommended-conf
+          - SecResponseBodyAccess Off         # Override to not process response bodies
+          - Include @crs-setup-conf           # Initialize the Core Rule Set
+          - Include @owasp_crs/*.conf         # Load the Core Rule Set
+          # Custom rule to deny requests with query parameter 'id=0'
+          - SecRule ARGS:id "@eq 0" "id:1,phase:1,deny,status:403,msg:'Invalid id',log,auditlog"
+          - SecRule REQUEST_HEADERS:user-agent "@contains curl" "id:2,pass,log,logdata:'someone used curl to access'"
```

Comment:
I agree with your previous comments we just can keep the line commented out 
```- SecRule REQUEST_HEADERS:user-agent "@contains curl" "id:2,pass,log,logdata:'someone used curl to access'"```

Reply:
I'd rather not add this rule. There is already one rule showing how to add custom rules, which is the objective. Why this rule and not any other that shows any other SecLang keyword? Let's keep the example focused on the CRS and showing the syntax.

Reply:
agree